### ISSUE TITLE
Error: no implicit conversion of String into Integer

### ISSUE THREAD

**Issue by jechol** [Homebrew/homebrew-bundle#824]
`brew bundle` failed with error.

1) command
```console
brew bundle --file=Brewfile --cleanup
```

2) output
```console
Please report this bug:
  https://github.com/Homebrew/homebrew-bundle/issues
/usr/local/Homebrew/Library/Taps/homebrew/homebrew-bundle/lib/bundle/locker.rb:92:in `block in brew_list_info'
/usr/local/Homebrew/Library/Taps/homebrew/homebrew-bundle/lib/bundle/locker.rb:91:in `each'
/usr/local/Homebrew/Library/Taps/homebrew/homebrew-bundle/lib/bundle/locker.rb:91:in `each_with_object'
/usr/local/Homebrew/Library/Taps/homebrew/homebrew-bundle/lib/bundle/locker.rb:91:in `brew_list_info'
/usr/local/Homebrew/Library/Taps/homebrew/homebrew-bundle/lib/bundle/locker.rb:44:in `block in lock'
/usr/local/Homebrew/Library/Taps/homebrew/homebrew-bundle/lib/bundle/locker.rb:36:in `each'
/usr/local/Homebrew/Library/Taps/homebrew/homebrew-bundle/lib/bundle/locker.rb:36:in `lock'
/usr/local/Homebrew/Library/Taps/homebrew/homebrew-bundle/lib/bundle/installer.rb:57:in `install'
/usr/local/Homebrew/Library/Taps/homebrew/homebrew-bundle/lib/bundle/commands/install.rb:10:in `run'
/usr/local/Homebrew/Library/Taps/homebrew/homebrew-bundle/cmd/bundle.rb:94:in `bundle'
/usr/local/Homebrew/Library/Homebrew/brew.rb:119:in `<main>'
```

**Comment by jacobbednarz** [comment]
I think this is a duplicate of #822 and fixed in #823. Can you try upgrading and running the command again?

**Comment by jechol** [comment]
This is fixed after `brew upgrade`. Thanks.
